src/library/scala/Array.scala

```diff
@@ -55,6 +55,19 @@ object Array {
    */
   def newBuilder[T](implicit t: ClassTag[T]): ArrayBuilder[T] = ArrayBuilder.make[T](t)
 
+  /** Builds an array from the iterable collection
+   *
+   *  {{{
+   *  scala> Array.from(Seq(1,5))
+   *  val res75: Array[Int] = Array(1, 5)
+   *
+   *  scala> Array.from(Range(1,5))
+   *  val res76: Array[Int] = Array(1, 2, 3, 4)
```

Comment:
I think you have to motivate `res75` and `res76`, which are like characters that suddenly appear in the late chapters of a novel. Also, it would be nice to write `Range(1, 5)` with a space. REPL adds a space for good reason.

It would be kind of cool if you read the scaladocs A-Z, you'd see the results in REPL examples as though entered contiguously, like bingeing all seven seasons of a tv show where you actually experience the story arcs.